auth/project_config_mgt.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"firebase.google.com/go/v4/internal"
+)
+
+// ProjectConfig represents the properties to update on the provided project config.
+type ProjectConfig struct {
+	MultiFactorConfig *MultiFactorConfig `json:"mfa,omitEmpty"`
+}
+
+func (base *baseClient) GetProjectConfig(ctx context.Context) (*ProjectConfig, error) {
+
+	req := &internal.Request{
+		Method: http.MethodGet,
+		URL:    base.projectMgtEndpoint,
+	}
+	var result ProjectConfig
+	if _, err := base.httpClient.DoAndUnmarshal(ctx, req, &result); err != nil {
```

Comment:
nit: We can clean this up a little by adding a reusable `makeRequest()` function if you would like. You can format the string URL (from above) in here or in that reusable `makeRequest()` function.